src/process/process/libs/scheduler/scheduler.cpp

```diff
@@ -1,3 +1,7 @@
 #include "scheduler.hpp"
 
-namespace scheduler {} // namespace scheduler
+namespace process {
```

Comment:
scheduler is a library to process, and should not be in the same namespace, so this namespace can be removed here.